configure.ac

```diff
@@ -672,6 +689,7 @@ echo "
         Python:                   ${PYTHON} (${PYTHON_VERSION})
         pylint:                   ${PYLINT}
         jslint:                   ${JSLINT}
+        rpmlint:                  $enable_rpmlint
```

Comment:
nit: shouldn't this follow the same pattern as the others, e.g. upper-case ${ENABLE_RPMLINT} ?

Reply:
Other than the nit the patch LGTM.

Reply:
PYLINT and JSLINT are shell variables in this case as well as enable_rpmlint. There is no common convention in IPA configure.ac regarding upper/low case(both exist). It doesn't matter for me. If you think the upper is more appropriate then I'll change it.

Reply:
That's true about inconsistency but most (I didn't check for all) of the optional/detected features use upper case. They universally use the ${<variable>} format.

Reply:
Thank you, pushed the 'upper' version.